uiplib/gui/settingsTab.py

```diff
@@ -147,6 +148,20 @@ def handle_settings(appObj):
     update_settings(appObj.new_settings)
 
 
+def update_settings_view(appObj, mainFrame, settings):
+    """Add and view default settings when Reset is pressed."""
+    appObj.pics_folder.set(settings['pics-folder'])
+    appObj.unsplash.set(True)
```

Comment:
hey so instead of defaulting using a constant variable there is a `check_sites` function in `utils.py` can you use that?